File: antilavado-service/src/main/java/ar/com/jf/antilavado/service/license/LicenseService.java
package ar.com.jf.antilavado.service.license;

import ar.com.jf.antilavado.repository.dto.response.license.LicenseData;

/**
 * LicenseService.java
 *
 * PLAYFT.
 *
 * Copyright (c) 2015 ****Fernando Valdes <fervaldes11@gmail.com>****
 *
 * Created by ktufernando on 25/02/2016.
 */
public interface LicenseService {

    LicenseData addLicense(String licenseString) throws Exception;
    LicenseData getLicenseData() throws Exception;

}

File: antilavado-web/src/main/webapp/modules/license/services/license.service.js
/**
 * license.service.js
 *
 * PLAYFT.
 *
 * Copyright (c) 2015 ****Fernando Valdes <fervaldes11@gmail.com>****
 *
 * Created by ktufernando on 29/02/2016.
 */

'use strict';

//Articles service used for communicating with the articles REST endpoints
angular.module(ApplicationConfiguration.applicationModuleName)

    .factory('Licenses', function ($resource, BASE_CONFIG) {
        return $resource(BASE_CONFIG.basePath + 'anonymous/license')
    });

